clang/tools/scan-build-py/lib/libear/ear.c

```diff
@@ -651,13 +651,14 @@ const unsigned char g_data[] = {
 // if ar_binary_used is zero, the content of file is filled with "/*emtpy-file*/"
 // if ar_binary_used is not zero,  the content of file is initialized by g_data[].
 static int generate_file(char *filename, int ar_binary_used) {
-  char buf[512];
-  char cmd[512];
+  char buf[PATH_MAX];
```

Comment:
Please check the default size of stack by :  ulimit -s,  may need unlift the limit by calling ulimit()

Reply:
On UB20/UB22, the output of `ulimit -s` is 8192 in units of 1024 bytes, so the tack size is 8MB, the current usage does go not out of bound.

Reply:
Please also check the Primiory OS we claim to support:  Sles sp3/4,  RHEL: 8, 9, ...

Reply:
Double checked, same behavior on Sel SP3/SP4,  RHEL8,9, Fed36/37, and UB20/UB22.